src/datachain/client/fsspec.py

```diff
@@ -364,6 +364,12 @@ def open_object(
         assert not file.location
         return FileWrapper(self.fs.open(self.get_full_path(file.path)), cb)  # type: ignore[return-value]
 
+    def upload(self, path: str, data: bytes) -> "File":
```

Comment:
why can't we add a tests here? or do you plan to rewrite it?

I know that it is a thin wrapper, but it still hits multiple apis + path, etc